cabbage/task_worker.py

```diff
@@ -28,7 +28,7 @@ def run(self, timeout: int = SOCKET_TIMEOUT) -> None:
                 break
 
             logger.debug("Waiting")
-            select.select(rlist=[connection], wlist=[], xlist=[], timeout=timeout)
+            select.select([connection], [], [], timeout)
```

Comment:
For information, I had to change this because this causes an error (not present in the tests) `TypeError: select() takes no keyword arguments`

Reply:
Oo ok, sorry about that :/

Reply:
This weren't triggered by the tests so there was no way to know

Reply:
Well, it tells something about our tests, doesn't it ? Can we update the acceptance test to do something ? Either defering a task from another task or spinning a thread that waits for a second and then creates a task should do the trick.

Reply:
I updated the acceptance tests to cover this (by using threads). It fails as expected with the call to `select()` we previously did.